modules/configuring-pipeline-run-workspaces.adoc

```diff
@@ -0,0 +1,142 @@
+:_module-type: CONCEPT
+
+[id="configuring-pipeline-run-workspaces_{context}"]
+= Configuring pipeline run workspaces
+
+[role='_abstract']
+The workspace feature in Data Science Pipelines Applications (DSPA) provides standardized ephemeral shared storage that lives for the duration of a pipeline run. This allows components within a pipeline to exchange large intermediate data without repeatedly uploading to object storage, improving performance and reducing object storage costs.
```

Comment:
It recently got renamed to AI Pipelines so you may have to rebase your PR and adjust the naming here.